source/administration/object-management.rst

```diff
@@ -59,16 +69,25 @@ However, hardware and network conditions may show performance impacts with large
 For a deeper discussion on the benefits of limiting prefix contents, see the article on :s3-docs:`optimizing S3 performance <optimizing-performance.html>`.
 
 Object Versioning
------------------
+----------------- 
 
 MinIO supports keeping multiple "versions" of an object in a single bucket. 
-Write operations which would normally overwrite an existing object instead result in the creation of a new versioned object. 
 
 .. image:: /images/retention/minio-versioning-multiple-versions.svg
    :alt: Object with Multiple Versions
    :align: center
 
-MinIO versioning protects from unintended overwrites and deletions while providing support for "undoing" a write operation.
+For buckets with versioning disabled or suspended, creating an object at a given namespace (``BUCKET/PREFIX/OBJECT.EXT``) overwrites the object permanently.
+Delete operations remove the object with no possibility of recovery ("hard delete").
+
+For buckets with versioning enabled, write operations on an existing object create a new version of that object using the write payload. 
+Delete operations produce a 0-byte "Delete Marker" as the latest version of the object ("soft delete").
```

Comment:
> Delete operations produce a 0-byte "Delete Marker" _for_ the latest version of the object.

I think "for" is better here than "as".

---

I feel like this paragraph and the one above might be easier to consume as a table, or maybe select h3s.
But maybe that's a different pass.

Reply:
Fixed, and took a pass at a table.